### ISSUE TITLE
Prediction in new image :Tutorial 9

### ISSUE THREAD

**Issue by alfredeep** [Hvass-Labs/TensorFlow-Tutorials#24]
Dear Hvass
 I am trying to test your tensorflow codes. I started by making a classification of my images and I applied all the steps in the tutorial 9 and all these good.

But the problem is that I do not know how to actually test it in other images are not in the folder trainnig test.
I want to evaluate the model by other images and that gives me percentage predictions like retraining in tensorflow examples. I give it an image and it tells me in which class it appertains and here is the code that I use it for the evaluation:
-------------------------------------------------
import tensorflow as tf
import sys

# change this as you see fit
image_path = sys.argv[1]

# Read in the image_data
image_data = tf.gfile.FastGFile(image_path, 'rb').read()

# Loads label file, strips off carriage return
label_lines = [line.rstrip() for line 
                   in tf.gfile.GFile("/home/ubuntu/labels.txt")]

# Unpersists graph from file
with tf.gfile.FastGFile("/home/ubuntu/graph.pb", 'rb') as f:
    graph_def = tf.GraphDef()
    graph_def.ParseFromString(f.read())
    _ = tf.import_graph_def(graph_def, name='')

with tf.Session() as sess:
    # Feed the image_data as input to the graph and get first prediction
    softmax_tensor = sess.graph.get_tensor_by_name('final_result:0')
    
    predictions = sess.run(softmax_tensor, \
             {'DecodeJpeg/contents:0': image_data})
    
    # Sort to show labels of first prediction in order of confidence
    top_k = predictions[0].argsort()[-len(predictions[0]):][::-1]
    
    for node_id in top_k:
        human_string = label_lines[node_id]
        score = predictions[0][node_id]
        print('%s (score = %.5f)' % (human_string, score))
---------------------------------

And here is the command that I use it: python3 test.py  image.jpg (New image)
Result:      0.87  C
                 0. 13 O

But this script uses the graph of the inception, but for our case in the tutorial it is registered "pkl" either the classes or the training.
Or, How to export my training model in graph.pb t to use the same script above to do the prediction just I will replace the inception graph with my cnn training graph

Thank you.

**Comment by Hvass-Labs** [comment]
This is not really a question about Tutorial 09. This is a more general question about how to do certain things in TensorFlow. These questions should always be asked on StackOverflow. You are more likely to get a good answer there - and it will also benefit many other people in the future who have similar problems.
